CRM/Core/BAO/SchemaHandler.php

```diff
@@ -295,7 +294,7 @@ public static function buildForeignKeySQL($params, $separator, $prefix, $tableNa
    *
    * @return bool
    */
-  public static function alterFieldSQL(&$params, $indexExist = FALSE, $triggerRebuild = TRUE) {
+  public static function alterFieldSQL($params, $indexExist = FALSE, $triggerRebuild = TRUE) {
 
     // lets suppress the required flag, since that can cause sql issue
     $params['required'] = FALSE;
```

Comment:
@eileenmcnaughton do we need to carry forward this assignment?

Reply:
The function that calls it (the only one) doesn't make use of the $params afterwards

Reply:
Yup make sense.